### PR TITLE
Add book backend

### PR DESCRIPTION
**Pull request by prazes**
Made a working add books page, connecting the frontend and backend. Successfully stored book data.

### CODE REVIEW

**Comment by sunnyxu2** [comment]
Looks good, but book router could use more work though.

**Comment by roya120** [comment]
Looks good.

File: client/src/components/addBook/AddBook.js [hunk 1]
```diff
@@ -26,21 +67,23 @@ return(
 
 
 <div class="container">
-<form> 
+<form action ='/' onSubmit={handleSubmit}>
```
Comment (arrennbaral) [inline client/src/components/addBook/AddBook.js:70]: So, once the request is submitted, it directs to the homepage?

Comment (arrennbaral) [reply client/src/components/addBook/AddBook.js:70]: Maybe direct it to the library page?

**Review by arrennbaral — APPROVED** [review]

File: client/src/components/addBook/AddBook.js (hunk 1 above)
Comment (prazes) [reply client/src/components/addBook/AddBook.js:70]: I just put it to homepage for now since we didn't have a page for isLogged  in.

**Review by sunnyxu2 — APPROVED** [review]
Retract previous comment about router; still looks good

**Pull request merged**